src/game.js

```diff
@@ -46,21 +46,21 @@ class Game {
   }
 
   checkForWin() {
-    if (this.evaluateTopRow()) {
+    if (this.evaluateForThree("topLeft", "topMiddle", "topRight")) {
```

Comment:
Have you come across [JavaScript Switch Statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)? This is another syntax that's useful when working with this number of `else if` conditions! I generally start building them from the `default` case outward (in this case, that sounds like your draw condition) and then adding on the additional conditions that trigger certain behavior (in this case, an `endGame()`).

Reply:
Thank you for your help with further refactoring this part. It was really helpful for me to have a chance to write out all of the logic step by step (even though it was repetitive), then refactor on my own to implement reusable functions, and then look over it with you to implement and array to better organize the different win possibilities.